packages/scaffold/src/views/w3m-email-verify-otp-view/index.ts

```diff
@@ -26,12 +26,26 @@ export class W3mEmailVerifyOtpView extends LitElement {
   // -- State & Properties -------------------------------- //
   @state() private loading = false
 
+  @state() private timeoutTimeLeft = 30
```

Comment:
Logic for email timeout already exists here in `wallet` package under helpers here
https://github.com/WalletConnect/web3modal/blob/V3/packages/wallet/src/W3mFrameHelpers.ts#L31

It's also more robust and based on `localStorage`. You should make a helper within core perhaps, that uses that value (if it exists) to bootstrap timers here.